CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-### 0.5.1 (Next)
+### 0.6.0 (12/13/2023)
```

Comment:
For now write "Next", we'll release and change it.

Reply:
sure, done